tests/test_learner_data.py

```diff
@@ -138,7 +138,7 @@ def test_learner_data_instructor_paced_w_certificate(self, mock_certificate_api,
         assert learner_data[0].course_id == self.course_id
         assert learner_data[0].course_completed
         assert learner_data[0].completed_timestamp == self.NOW_TIMESTAMP
-        assert learner_data[0].grade == 'A-'
+        assert learner_data[0].grade == 'Pass'
```

Comment:
Is it possible to replace this string with the enum reference?  That way if "Pass" ever changes to "Passing" we won't have to update the tests.